azurelinuxagent/common/osutil/default.py

```diff
@@ -299,12 +301,17 @@ def is_current_instance_id(self, id_that):
     @staticmethod
     def is_cgroups_supported():
         """
-        Enabled by default; disabled in WSL/Travis
+        Enabled by default; disabled in WSL/Travis (Disabled on Trusty only)
         """
         is_wsl = '-Microsoft-' in platform.platform()
-        is_travis = 'TRAVIS' in os.environ and os.environ['TRAVIS'] == 'true'
+        supported = True  # By default supported in Travis
+
+        # Travis with python 2.6 fails (on Trusty based systems)
+        if PY_VERSION_MAJOR == 2 and PY_VERSION_MINOR == 6:
```

Comment:
this test should be based on the platform (trusty) rather than on the python version